plugins/ksp/testData/api/typeParameterReference.kt

```diff
@@ -0,0 +1,17 @@
+// WITH_RUNTIME
+// TEST PROCESSOR: TypeParameterReferenceProcessor
+// EXPECTED:
+// Foo.T1
```

Comment:
NIt: It would be nice if you can add the names of the symbols for the type references. Seems not easy with your current test processor though.

Reply:
That's why the references are named differently :)